AGPs/AGP-1H1.md

```diff
@@ -0,0 +1,44 @@
+---
+AGP: tbd
+Title: Conviction Voting + Template
+Author: @lkngtn (@1hive)
+Status: Stage II
+Track: Finance
+Created: 2019-11-12
+---
+
+# **AGP-TBD: Conviction Voting + Template**
+
+## **Address of the transfer recipient**
+
+ `0x41ecc492719a9cd750f8c9a693896d0dd621d4ec` Vault of the [conviction project organization](https://mainnet.aragon.org/#/conviction
+
+## **Amount of the transfer**
+
+75,000 DAI Due upon approval
+25,000 ANT Due upon successful delivery
+
+## **Number and frequency of transfers if recurring**
+
+2 (One upon approval, one upon successful delivery).
+For the purpose of budgeting, the full amount should be debited against this quarters budget.
+
+## **Purpose of the transfer**
+
+We believe that one of most significant blockers of wider adoption of Aragon is the lack of truly public, permissionless organizations. Organizations that anyone can join and participate in immediately without asking, while at the same time mitigating known deficiencies with participation and capture associated with fund allocation in public communities.
```

Comment:
It remains unclear how Fundraising + Conviction Voting + Escrow + Dandelion Voting can make permissionless organizations. We should elaborate a little bit on that.